### PR TITLE
Ci profiles

### PR DESCRIPTION
**Pull request by amalykhi**
Task https://issues.redhat.com/browse/OCM-2799
Added cluster profiles from OCM backend matrix
aws version is changed to be updated
default env for ci is updated to staging

### CODE REVIEW

File: tests/ci/profiles/tf_cluster_profile.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    autoscale: true
+    byok: true
+    version: "latest"
+    proxy: true
+    labeling: true
+    tagging: true
+    channel_group: staging
+    zones: "a,b,c"
+    imdsv2: "required"
+    oidc_config: UnManaged
```
Comment (xueli181114) [inline tests/ci/profiles/tf_cluster_profile.yml:50]: Low the case of oidc_config to `un-managed`

**Review by nirarg — APPROVED** [review]
Those files are not related to this PR:
* ci/e2e/account_roles_files/variables.tf
* examples/create_rosa_sts_cluster/oidc_configuration/cluster_with_unmanaged_oidc_config/variables.tf

Its ok to merge with those also
But for next PRs please make sure to separate those to different PRs

**Pull request merged**